src/com/theironyard/novauc/Main.java

```diff
@@ -1,8 +1,236 @@
 package com.theironyard.novauc;
 
+import org.h2.tools.Server;
+import spark.ModelAndView;
+import spark.Session;
+import spark.Spark;
+import spark.template.mustache.MustacheTemplateEngine;
+
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.HashMap;
+
 public class Main {
+static HashMap<String, User> users = new HashMap<>();
+static ArrayList<Restaurant> restaurants = new ArrayList<>();
+
+
+
+
+    public static ArrayList<Restaurant> selectResults(Connection conn) throws SQLException {
+        ArrayList<Restaurant> restaurants = new ArrayList<>();
+        PreparedStatement stmt = conn.prepareStatement("SELECT FROM restaurants INNER  JOIN users ON restaurants.user_id = users.id WHERE restaurants = ?");
+        ResultSet results = stmt.executeQuery();
```

Comment:
This does not run. You have three errors in your SQL join queries.  Fix those first so you have a running program.